modules/ROOT/edits/enable-inbound-traffic-self.adoc

```diff
@@ -0,0 +1,22 @@
+= Configure Domains and Inbound Traffic (On Self-Managed Kuberenetes)
+
```

Comment:
GLOBALLY introduce the product by its full name at first use in an .adoc file, and then be sure to use the appropriate abbreviations, making clear whether the context is for "plain" RTF or RTF on Kubernetes AKS or EKS.

Reply:
Also, ditto/GLOBAL previous question about ingress vs. Ingress.